src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -466,6 +466,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Numerics\Matrix4x4.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Numerics\Plane.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Numerics\Quaternion.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Numerics\Register.cs" />
```

Comment:
Why do we still need Register.cs ?

Reply:
> I had to add back Register. ARM64 was failing due to some disconnect between the HFA classification causing it to be `hasMultiregStructArgs` and `fgMorphMultiregStructArgs` not reporting the type as TYP_STRUCT

I don't want to block the overall simplification on this and am going to do some deeper investigation as to why. @CarolEidt might also have some insight as I believe she added much of the HFA support.

Reply:
ok, I agree it is fine to look at this separately.

Reply:
It's not immediately apparent to me what the issue would be with the HFA classification, but agree that it would be best to address that separately.